src/content/docs/zh-cn/install/manual.mdx

```diff
@@ -112,26 +144,49 @@ export default defineConfig({});
 如果你想集成像 React、Svelte 这样的[UI 框架组件](/zh-cn/core-concepts/framework-components/)或是使用其他类似 Tailwind 或 Partytown 这样的工具，你可以在[手动导入并配置集成](/zh-cn/guides/integrations-guide/)章节内获取更多信息。
 
 
-📚 阅读Astro的[API配置引用](/zh-cn/reference/configuration-reference/)章节可以获得更多内容。
+📚 阅 读Astro 的[API配置参考](/zh-cn/reference/configuration-reference/)章节可以获得更多内容。
 
+## 6. 添加 TypeScript 支持
 
-## 6. 接下来
+TypeScript 使用`tsconfig.json`进行配置。即使你不编写 TypeScript 代码，这个文件也很重要，因为 Astro 和 VS Code 等工具知道如何理解你的项目。 如果没有 `tsconfig.json` 文件，编辑器将不完全支持某些功能（如 npm 包导入）。
 
-如果你按上述一步步操作，你的项目目录应该看上去像是这样：
+如果你打算编写 TypeScript 代码，建议使用 Astro 的`strict`或`strictest`的模板。 你可以在 [astro/tsconfigs/](https://github.com/withastro/astro/blob/main/packages/astro/tsconfigs/) 查看和比较三个模板配置。
+
+在项目的根目录创建 `tsconfig.json`，并将下面的代码复制到其中。（你可以为 TypeScript 模板使用 `base`、`strict` 或 `strictest`）：
 
+```json title="tsconfig.json" "base"
+{
+  "extends": "astro/tsconfigs/base",
+}
 ```
-├── node_modules/
-├── src/
-│   └── pages/
-│   │   └── index.astro
-├── public/
-│   ├── robots.txt
-├── astro.config.mjs
-├── package.json
-└── package-lock.json (or: yarn.lock, pnpm-lock.yaml, etc.)
+
+最后，创建`src/env.d.ts`让 TypeScript 知道 Astro 项目中可用的环境类型：
+
+```ts title="src/env.d.ts"
+/// <reference types="astro/client" />
 ```
 
+📚 阅读 Astro 的 [TypeScript 设置指南](/zh-cn/guides/typescript/#设置)以获取更多信息。
+
+## 7. 接下来
+
+如果你按上述一步步操作，你的项目目录应该看上去像是这样：
+
+<FileTree>
+- node_modules/
+- public/
+  - robots.txt
+- src/
+  - pages/
+    - index.astro
+  - env.d.ts
+- astro.config.mjs
+- package-lock.json or `yarn.lock`, `pnpm-lock.yaml`, etc.
```

Comment:
This part (or `yarn.lock`, `pnpm-lock.yaml`, etc.) can be translated!

Reply:
Thanks!